src/parallax/fourth/sections/three.module.scss

```diff
@@ -0,0 +1,41 @@
+.section3 {
+  overflow-y: hidden;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  background: url("/assets/p4/background3.png");
+  background-size: cover;
+  color: white;
+
+  h2 {
+    text-transform: uppercase;
+    font-size: 3.2vw;
+  }
+
+  p {
+    font-size: 1.3rem;
+    margin-bottom: 40px;
+  }
+
+  &Gallery {
+    display: flex;
+    gap: 50px;
+    justify-content: center;
+  }
+
+  // Animation
+  .fadeIn {
+    opacity: 0;
+    transition-property: transform, opacity;
+    transition-duration: 1s;
+    transition-timing-function: linear;
```

Comment:
todo: Sandrina's animation rule nr 1: never ever use `linear`, it's so unnatural xD

To show use `ease-out`. To hide it's always `ease-in` (or variations of those two presets). [reference](https://css-tricks.com/ease-out-in-ease-in-out).